### PR TITLE
[18.0][MIG] base_cron_exclusion: Migration to 18.0

### PR DESCRIPTION
**Pull request by xaviedoanhduy**

### CODE REVIEW

File: base_cron_exclusion/models/ir_cron.py [hunk 1]
```diff
@@ -67,5 +67,5 @@ def _process_job(cls, db, cron_cr, job):
             res = super()._process_job(db, cron_cr, job)
         finally:
             locked_crons.close()
-            _logger.debug("released blocks for cron job %s" % job["cron_name"])
+            _logger.debug(f"released blocks for cron job {job['cron_name']}")
```
Comment (sebalix) [inline base_cron_exclusion/models/ir_cron.py]: Better to use the parameters of logger methods for the interpolation:
```suggestion
            _logger.debug("released blocks for cron job %s", job['cron_name'])
```

**Review by sebalix — APPROVED** [review]
Small remark, LG

File: base_cron_exclusion/models/ir_cron.py (hunk 1 above)
Comment (xaviedoanhduy) [reply base_cron_exclusion/models/ir_cron.py]: Hello @sebalix, 
thank for your suggestions.

**Review by TDu — APPROVED** [review]
:+1:  For adding the unit tests

**Review by houssine78 — APPROVED** [review]
LGTM

**Comment by houssine78** [comment]
@pedrobaeza this PR looks like it can be merged :) thanks !

**Pull request merged**